backend/wmg/cube_creation/load_cube.py

```diff
@@ -121,11 +122,17 @@ def recurse(node):
     mapping = {}
     for tissue, cell_df in cell_type_by_tissue.items():
         cells = list(cell_df)
-        ordered_cells = compute_ordering(cells, "CL:0000003") # TODO: is this the right root?
+        ordered_cells = compute_ordering(cells, "CL:0000003")  # TODO: is this the right root?
         mapping[tissue] = ordered_cells
 
-    with open("ordered-cells.json", "w") as f:
-        json.dump(mapping, f)
+    data = []
+    for tissue, cells in mapping.items():
+        for cell in cells:
+            data.append((tissue, cell))
+
+    df = pd.DataFrame(data, columns=["tissue", "cell"])
```

Comment:
columns will need to match [this structure](https://github.com/chanzuckerberg/single-cell-data-portal/blob/2e4e129106aaec6dba7431eac7b8044a415c0e6f/tests/unit/backend/wmg/fixtures/snapshot.py#L150-L154), with those precise column names.  The `order` column can be added with just this change:
```
        for i, cell in enumerate(cells):
            data.append((tissue, cell, i))
```